apps/fulfillment-e2e/src/integration/filters.cy.ts

```diff
@@ -0,0 +1,83 @@
+import { FiltersFragment } from '../support/page_fragments/filters.fragment';
+import { PickingListsFragment } from '../support/page_fragments/picking-lists.fragment';
+
+const pickingListsFragment = new PickingListsFragment();
+const filtersFragment = new FiltersFragment();
+
+describe('When user opens the filters', () => {
+  const largeAndEarliestOrderFirst = (exists = true) =>
```

Comment:
```
//should not change the order
largeAndEarliestOrderFirst();
```

can we rename it to show exactly what it does? like `checkOrderSorting(isChanged: boolean)` or something?

Also, we can move this function out from the `describe` block to make it visible to everyone